pkg/block/s3/adapter.go

```diff
@@ -441,6 +443,15 @@ func (a *Adapter) GetPreSignedURL(ctx context.Context, obj block.ObjectPointer,
 			Bucket: aws.String(bucket),
 			Key:    aws.String(key),
 		}
+		// Add content-disposition if filename provided
+		if filename != "" {
+			contentDisposition := mime.FormatMediaType("attachment", map[string]string{
+				"filename": path.Base(filename),
+			})
+			if contentDisposition != "" {
```

Comment:
Not sure if this is a formatting thing in github bit how can it be empty at this point?

Reply:
Any issue in FormatMediaType will return an empty string, in the s3 case the member is pointer which I didn't want to set to empty string.